bindings/python/native/iota_client/client.py

```diff
@@ -84,3 +155,34 @@ def unsynced_nodes(self):
         """Returns the unsynced nodes.
         """
         return self.call_client_method('UnsyncedNodes')
+
+    def prepare_transaction(self, secret_manager=None, options=None):
+        """Prepare a transaction for signing.
+        """
+        return self.call_client_method('PrepareTransaction', {
+            'secret_manager': secret_manager,
+            'options': options
+        })
+
+    def sign_transaction(self, secret_manager, prepared_transaction_data):
+        """Sign a transaction.
+        """
+        return self.call_client_method('SignTransaction', {
+            'secret_manager': secret_manager,
+            'prepared_transaction_data': prepared_transaction_data
+        })
+
+    def store_mnemonic(self, secret_manager, mnemonic):
+        """Store a mnemonic in the Stronghold vault.
+        """
+        return self.call_client_method('StoreMnemonic', {
+            'secret_manager': secret_manager,
+            'mnemonic': mnemonic
+        })
+
+    def submit_payload(self, payload_dto):
+        """Store a mnemonic in the Stronghold vault.
```

Comment:
Wrong description

Reply:
Thanks for pointing this out. Modified at [45d47b3](https://github.com/iotaledger/iota.rs/pull/1055/commits/45d47b33738532b58b0b0f52951dd794dc8954f9)